watsononlinestore/database/cloudant_online_store.py

```diff
@@ -183,3 +183,21 @@ def add_doc_if_not_exists(self, doc, unique_property_name):
                 LOG.exception("Cloudant DB exception:")
             finally:
                 self.client.disconnect()
+
+    def optimize_cloudant_url(url):
+        """if the url is in the pattern
```

Comment:
super minor optimization, you *could* only run this if it's py3 by adding a guard like:

```
import sys
if sys.version_info[0] < 3:
    return url
...
# rest of the code
```